QUALITY_DECLARATION.md

```diff
@@ -33,11 +33,11 @@ All installed headers are in the `include` directory of the package, headers in
 
 ## Change Control Process [2]
 
-`class_loader` follows the recommended guidelines for ROS Core packages in the [ROS 2 Developer Guide](https://index.ros.org/doc/ros2/Contributing/Developer-Guide/#change_control_process).
+`class_loader` follows the recommended guidelines for ROS Core packages in the [ROS 2 Developer Guide](https://docs.ros.org/en/foxy/Contributing/Developer-Guide.html#change-control-process).
 
 ### Change Requests [2.i]
 
-This package requires that all changes occur through a pull request. Check the [ROS 2 Developer Guide](https://index.ros.org/doc/ros2/Contributing/Developer-Guide/#pull-requests) for additional information.
+This package requires that all changes occur through a pull request. Check the [ROS 2 Developer Guide](https://docs.ros.org/en/foxy/Contributing/Developer-Guide.html#pull-requests) for additional information.
```

Comment:
@clalancette why not `rolling` everywhere? There's a `foxy` branch in this repo for stable, released code.

Reply:
Because I was following links, and forgot to update this one :).  Will fix.

Reply:
Should be fixed by f71f686.